source/includes/_hoist.md

```diff
@@ -0,0 +1,55 @@
+# Hoist
+
+```javascript
+1.
+var name = "Warren";
+//Before Hoist
+
+var name; // This is the declaration
+name = "Warren"; //This is the assignment
+//After Hoist
+
+```
+
+
+
+```javascript
+ 2.
+function parkour(){} //Hoisted function declaration
+function iDeclareBankruptcy(){} //Hoisted function declaration
+
+var parkour; //Hoisted variable declaration
+var iDeclareBankruptcy(){} //Hoisted variable declaration
+
+parkour = function(){} // Function expression
+iDeclareBankruptcy = function(){} // Function expression
```

Comment:
I think this section would be ore useful if the 'before' and 'after' were more obvious. I found it confusing.